src/peerjs/ext/http-proxy.py

```diff
@@ -162,9 +162,6 @@ async def peer_disconnected(peerId):
                       'Resetting to last known ID.')
             peer._id = savedPeerId
         peer._lastServerId = savedPeerId
-        global _is_shutting_down
-        if not _is_shutting_down:
-            await peer.reconnect()
```

Comment:
Reconnecting while peer is being destroyed causes an endless loop. Leave reconnect to the make_discoverable() coro.